frontend/simple/main.js

```diff
@@ -10,9 +10,17 @@ import store from './js/state'
 async function loadLastUser () {
   let user = await db.loadCurrentUser()
   if (user) {
-    // TODO: handle error if we fail to find such a user
-    let identityContractId = await namespace.lookup(user)
-    await store.dispatch('login', {name: user, identityContractId})
+    let identityContractId = await namespace.lookup(user).catch((err) => {
```

Comment:
This await + catch is confusing, because it mixes Promise-style code with async/await-style code, making it unclear what the value of `identityContractId` will actually be.

So I recommend that, even if this works, convert to a try/catch block, which is what async/await should be paired with.

Reply:
you are right, this does look confusing! try/catch version coming up